---
layout: page-fullwidth
title: "Haoyang's paper on regulatory variant prioritization is published in Bioinformatics"
teaser: "GERV: a statistical method for generative evaluation of regulatory variants for transcription factor binding"
subheadline: "Functional Genomics"
header: no
image:
    thumb:  gerv-img.jpg
    homepage: gerv-img.jpg
categories:
    - research
permalink: /research/gerv/
---


The majority of disease-associated variants identified in genome-wide association studies reside in noncoding regions of the genome with regulatory roles. Thus being able to interpret the functional consequence of a variant is essential for identifying causal variants in the analysis of genome-wide association studies. We present GERV (generative evaluation of regulatory variants), a novel computational method for predicting regulatory variants that affect transcription factor binding. GERV learns a k-mer-based generative model of transcription factor binding from ChIP-seq and DNase-seq data, and scores variants by computing the change of predicted ChIP-seq reads between the reference and alternate allele. The k-mers learned by GERV capture more sequence determinants of transcription factor binding than a motif-based approach alone, including both a transcription factor's canonical motif and associated co-factor motifs. We show that GERV outperforms existing methods in predicting single-nucleotide polymorphisms associated with allele-specific binding. GERV correctly predicts a validated causal variant among linked single-nucleotide polymorphisms and prioritizes the variants previously reported to modulate the binding of FOXA1 in breast cancer cell lines. Thus, GERV provides a powerful approach for functionally annotating and prioritizing causal variants for experimental follow-up analysis.additional missing heritability.

![alt text]({{site.url}}/images/gerv-img.jpg)