---
title: "Identification of determinants of differential chromatin accessibility through a massively parallel genome-integrated reporter assay"
authors: Jennifer Hammelman, Konstantin Krismer, Budhaditya Banerjee, David K. Gifford and Richard I. Sherwood
journal: "Genome Research"
journalspecific:  "2020"
doi: "10.1101/gr.263228.120"
arxiv: ""
ext: ""
img: Hammelman2020.png
date: "2020-09-24"
year: "2020"
---

A key mechanism in cellular regulation is the ability of the transcriptional machinery to physically access DNA. Transcription factors interact with DNA to alter the accessibility of chromatin, which enables changes to gene expression during development or disease or as a response to environmental stimuli. However, the regulation of DNA accessibility via the recruitment of transcription factors is difficult to study in the context of the native genome because every genomic site is distinct in multiple ways. Here we introduce the multiplexed integrated accessibility assay (MIAA), an assay that measures chromatin accessibility of synthetic oligonucleotide sequence libraries integrated into a controlled genomic context with low native accessibility. We apply MIAA to measure the effects of sequence motifs on cell type–specific accessibility between mouse embryonic stem cells and embryonic stem cell–derived definitive endoderm cells, screening 7905 distinct DNA sequences. MIAA recapitulates differential accessibility patterns of 100-nt sequences derived from natively differential genomic regions, identifying E-box motifs common to epithelial–mesenchymal transition driver transcription factors in stem cell–specific accessible regions that become repressed in endoderm. We show that a single binding motif for a key regulatory transcription factor is sufficient to open chromatin, and classify sets of stem cell–specific, endoderm-specific, and shared accessibility-modifying transcription factor motifs. We also show that overexpression of two definitive endoderm transcription factors, T and Foxa2, results in changes to accessibility in DNA sequences containing their respective DNA-binding motifs and identify preferential motif arrangements that influence accessibility.
