---
layout: page-fullwidth
title: "Software"
header:
     image_fullwidth: "stata0.jpg"
     caption: (image credit)
     caption_url: https://www.flickr.com/photos/justinjensen/4454664922/
permalink: "/software/"
---


More software can be found in our [GitHub](https://github.com/gifford-lab/) Repository.

   * [EC2-launcher](https://github.com/gifford-lab/ec2-launcher-pro) - A tool that lauches docker jobs on Amazon EC2. 
   
   * [Caffe-cnn](https://github.com/gifford-lab/mri-wrapper) - A [Caffe](http://caffe.berkeleyvision.org/)-based deep learning platform with automatic hyper-parameter searching.
    
   * [GERV](http://gerv.csail.mit.edu/) - A statistical method for predicting non-coding variants' effect on transcription factor binding.  
   
   
   * [PIQ](http://piq.csail.mit.edu/) - A computational method that identifies the genomic binding sites of hundreds of transcription factors (TFs) at corresponding motifs from DNase-Seq experiments with accuracy comparable to ChIP-Seq.
   
   * [GERM](https://github.com/christopherreeder/germ) - An approach to identifying conditional occupancy events for a protein from ChIA-PET data.
   
   * [SRPOUT](https://github.com/gifford-lab/SPROUT) - High Resolution Modeling of Chromatin Interactions.
   
   * [Fixseq](https://bitbucket.org/thashim/fixseq) - A over-dispersion correction technique that serves as a smarter way to de-duplicate counts. 
   
   * [Multipool](https://github.com/matted/multipool) - High-resolution genetic mapping for pooled sequencing
	
   * [GEM](http://groups.csail.mit.edu/cgs/gem/) - Integrative Genome wide Event finding and Motif discovery for ChIP-seq and ChIP-exo data.  
   
   * [GPS](http://groups.csail.mit.edu/cgs/onePageGPS/) - The Genome Positioning System for discovering binding events in ChIP-seq data.
   
   * [GSE](http://groups.csail.mit.edu/cgs/gse.html) - Genomic Spatial Events system, a database for storing, visualizing, and analyzing ChIP-base transcription factor binding data and gene expression data.
   
   * [GRAM](http://groups.csail.mit.edu/cgs/onePageGram/) - The GRAM (Genetic RegulAtory Modules) algorithm identifies modules, collections of genes that share common regulators, as well as expression profiles.
   
   * [JBD](http://groups.csail.mit.edu/cgs/jbd.html) - The JBD (Joint Binding Deconvolution) algorithm detects binding sites based on ChIP-chip data and guides.
   
   * [GeneProgram](http://groups.csail.mit.edu/cgs/geneprogram.html) - Automated discovery of functional generality of gene expression programs (sets co-expressed genes)

